### ISSUE TITLE
DOS stuck in channel -1

### ISSUE THREAD

**Issue by pott4t0** [v1s1t0r1sh3r3/airgeddon#503]
#### What is your airgeddon version?

v11.01

#### What is your Linux O.S. and which version?

Kali Linux 2022.2

#### Which is the chipset of your wireless card?

Intel Corporation Wireless 8265 / 8275 
(Not really sure about that but it was the only thing I could find using `lspci` and `sudo lshw -C network`)

#### Describe the issue and the steps to reproduce it

In evil twin attacks the dos window tries to find the AP on channel -1 instead of the actual channel the AP is at so it ends up not getting any beacon frames to start the deauthentication, also if I try to run airodump on another terminal during the attack it is also stuck in channel -1. I'd like to note here that when I run the script in dos pursuit mode with a second wifi card everything seems to work flawlessly. 
To sum up, I don't think that that's a driver issue because I can perform other similar attacks like getting handshakes without any problems and the wifi card supports virtual interfaces so I don't understand what could be causing this or if there are any workarounds to the issue.
![deauth_window](https://user-images.githubusercontent.com/109975566/180829549-33d39d9d-9e13-4d33-9b39-51c89be4f3e2.png)

**Comment by cjb900** [comment]
The AP you are attacking is most likely changing the channel it on when deauthing. this would explain why using DOS pursuit works but not using it doesn't and results in that channel -1. there is no fix for this unless you have access to the admin page for the AP you are attacking and can disable this feature.

**Comment by pott4t0** [comment]
> The AP you are attacking is most likely changing the channel it on when deauthing. this would explain why using DOS pursuit works but not using it doesn't and results in that channel -1. there is no fix for this unless you have access to the admin page for the AP you are attacking and can disable this feature.

In fact yeah that's my router and I have it fixed on channel 6.
Also if this was the case I shouldn't be able to get handshakes or DOS the AP with aireplay-ng either so the problem most likely doesn't have anything to do with channel hopping.

**Comment by OscarAkaElvis** [comment]
If you want to fight channel hopping of the target AP you can use DoS pursuit mode: https://github.com/v1s1t0r1sh3r3/airgeddon/wiki/FAQ%20&%20Troubleshooting#what-is-dos-pursuit-mode-exactly

Regarding DoS effectivity, read this: https://github.com/v1s1t0r1sh3r3/airgeddon/wiki/FAQ%20&%20Troubleshooting#dos-deauth-is-not-working-what-can-i-do
